packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -482,8 +482,7 @@ If you prefer video documentation, consider: https://www.youtube.com/c/flutterde
 
 In order to run your $application, type:
 
-  \$ cd $relativeAppPath
-  \$ flutter run
+  ${relativeAppPath != "." ? "\$ cd $relativeAppPath\n  \$ flutter run" : "\$ flutter run"}
```

Comment:
Can you do the logic here in an if statement setting a String variable before line 477? This is very difficult to follow the logic inline. Also you'll need to add a test. Actually, if you make this logic in a helper function that takes in the `relativeAppPath` as a parameter, you can unit test it, rather than having to write an expensive integration test.

Reply:
Sure! Will try to update tests and if not, then I'll get in touch on Discord

Reply:
@christopherfujino I extracted the logic into a separate function but I actually have no idea how I can add tests for this. Can you show me file that maybe already contains a unit test for the `create` command so that I can make changes in that?

Reply:
I would create a new file here: https://github.com/flutter/flutter/tree/master/packages/flutter_tools/test/general.shard/commands called `create_test.dart`. You can copy some boilerplate from the similar file: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/general.shard/commands/analyze_base_test.dart. Most importantly, it needs to have the license comment at the top.